.github/workflows/tests.yml

```diff
@@ -16,16 +16,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4]
+        php: [7.4, 8.0]
         mysql: [5.7]
+        sylius: ["~1.8.0", "~1.9.0", "v1.10.0-beta.1"]
+        exclude:
+          - php: 8.0
+            sylius: "~1.8.0"
+          - php: 8.0
+            sylius: "~1.9.0"
+
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: Switch PHP
-        run: |
-          sudo update-alternatives --set php /usr/bin/php${{ matrix.php }}
-          php -v
+      -
+        name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+            php-version: "${{ matrix.php }}"
+            extensions: intl
+            tools: symfony
+            coverage: none
```

Comment:
Any reason why you are changing this?

PHP 8.0 is installed on the runner. See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-software.

Reply:
https://github.com/TonySma/SyliusSettingsPlugin/runs/2906412101?check_suite_focus=true
Strangely GH seems to remove ".0" of 8.0 and it fail when alternative update. 

I just pick the Sylius method instead: 
https://github.com/Sylius/Sylius/blob/master/.github/workflows/packages.yml#L61

Reply:
Fixed with quotes